src/components/PostDetail.js

```diff
@@ -55,45 +51,78 @@ const styles = StyleSheet.create({
 const PostDetail = React.createClass({
   propTypes: {
     post: postPropTypes.isRequired,
+    navigator: postPropTypes.isRequired,
   },
 
-  render: function() {
+  getHTML: function() {
     const post = this.props.post;
-    let caption;
-    if (post.images.length > 0 && post.images[0].caption !== '') {
-      caption = (
-        <Text style={styles.caption}>{post.images[0].caption}</Text>
-      );
-    }
-    let byline;
-    if (post.authors) {
-      byline = (
-        <Text style={styles.byline}>By {post.authors.join(', ')}</Text>
-      );
+
+    const caption = (post.images.length > 0 && post.images[0].caption !== '') ? post.images[0].caption : '';
+    const author = (post.authors) ? `<p id='byline'>By ${post.authors.join(', ')} </p>` : '';
```

Comment:
Is this supposed to be `post.authors.length > 0`? [Empty arrays are truthy in javascript](https://developer.mozilla.org/en-US/docs/Glossary/Truthy).